### PR TITLE
Git_Basics.rst: Add a note

### PR DESCRIPTION
**Pull request by gopalgoel**
The note tells a newcomer to look for more commands in their favourite search engines.

Closes https://github.com/coala/coala/issues/3971

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by gopalgoel** [comment]
@rohanraju97 Yeah okay. Should I add that link and amend old commit keeping a single commit? Or adding the link part should come as a new commit?

**Comment by rohanraju97** [comment]
Keep it as a single commit!

**Comment by rohanraju97** [comment]
ack 6820609954e35388608e42fd5b5c0a2e15ce084b

File: docs/Developers/Git_Basics.rst [hunk 1]
```diff
@@ -453,3 +453,7 @@ use an interactive rebasing session. This opens an editor where you can enter
 commands (described below) for each commit to be rebased. These commands
 determine how individual commits will be transferred to the new base. You can
 also reorder the commit listing to change the order of the commits themselves.
+
+If you would like more information/commands, please use your favourite search
+engine to look for it. You could also read the
```
Comment (aptrishu) [inline docs/Developers/Git_Basics.rst]: You could also read this "git documentation" to learn more.
link the quoted part.

Comment (aptrishu) [inline docs/Developers/Git_Basics.rst]: Just don't need official tag , plus its not git doc actually.

**Comment by aptrishu** [comment]
unack 6820609954e35388608e42fd5b5c0a2e15ce084b

File: docs/Developers/Git_Basics.rst [hunk 2]
```diff
@@ -453,3 +453,7 @@ use an interactive rebasing session. This opens an editor where you can enter
 commands (described below) for each commit to be rebased. These commands
 determine how individual commits will be transferred to the new base. You can
 also reorder the commit listing to change the order of the commits themselves.
+
+If you would like more information/commands, please use your favourite search
+engine to look for it. You could also read this
```
Comment (Madhukar-SH) [inline docs/Developers/Git_Basics.rst]: Remove this. `You could also read git documentation to learn more` is better :)

File: docs/Developers/Git_Basics.rst [hunk 3]
```diff
@@ -453,3 +453,7 @@ use an interactive rebasing session. This opens an editor where you can enter
 commands (described below) for each commit to be rebased. These commands
 determine how individual commits will be transferred to the new base. You can
 also reorder the commit listing to change the order of the commits themselves.
+
+If you would like more information/commands, please use your favourite search
```
Comment (NiklasMM) [inline docs/Developers/Git_Basics.rst]: I'd add one more sentence explaining how easy it is to google for git stuff: "Git is widely used throughout the world and there are many good tutorials and git related Q&A threads out there."

File: docs/Developers/Git_Basics.rst [hunk 4]
```diff
@@ -453,3 +453,9 @@ use an interactive rebasing session. This opens an editor where you can enter
...
+engine to look for it. You could also read the
+`git documentation <https://git-scm.com/doc>`_ to learn more. Git is widely used
```
Comment (Nosferatul) [inline docs/Developers/Git_Basics.rst]: I'm not sure if we need the part with `Git is widely...`

Comment (NiklasMM) [inline docs/Developers/Git_Basics.rst]: Well, they added exactly what I suggested [here](https://github.com/coala/coala/pull/3980/files#r107846863). I find it beneficial to stress just how trivial it is to google git related stuff to keep people from asking them in the chatrooms.

Comment (supergr35) [inline docs/Developers/Git_Basics.rst]: I disagree with the addition of a link. The point of this issue was to enable the user to look for things on their own, thus I don't think we should hand them a link to what they're looking for. Making them search for something teaches them that they need to search for things. They won't search for anything if we give them a link.
Opinions?

Comment (gopalgoel) [inline docs/Developers/Git_Basics.rst]: I disagree. They would still need to search for commands even if they have this doc. It is not a cake walk once you visit the website. The point of providing the link is to suggest that it is one of the most authentic places to look for git commands.

Comment (NiklasMM) [inline docs/Developers/Git_Basics.rst]: I'd also vote for leaving the link in as it's a good starting point to get to know git basics and does not eliminate the need to look for things entirely.

**Comment by aptrishu** [comment]
ack 2bd31daab785d90298749bdf8aea9783ede2b13e

**Comment by aptrishu** [comment]
needs rebase.

**Comment by NiklasMM** [comment]
reack b9f2518

**Comment by NiklasMM** [comment]
@rultor merge

**Pull request merged**